org.gnome.EasyTAG.yaml

```diff
@@ -0,0 +1,141 @@
+      - type: patch
+        path: libid3tag-0.15.1b-fix_overflow.patch
+        strip-components: 0
+      - type: patch
+        path: libid3tag-0.15.1b-handle-unknown-encoding.patch
+      - type: patch
+        path: libid3tag-0.15.1b-id3v1-zero-padding.patch
+      - type: patch
+        path: libid3tag-0.15.1b-id3v2-endless-loop.patch
+        strip-components: 0
+      - type: patch
+        path: libid3tag-0.15.1b-pkgconfig.patch
+      - type: patch
+        path: libid3tag-0.15.1b-gperf-size_t.patch
+      - type: script
+        commands:
+          - autoreconf --force --install
+
+  - name: opus
+    cleanup:
+      - /share/aclocal
+    config-opts:
+      - --disable-doc
+      - --disable-extra-programs
+    sources:
+      - type: archive
+        url: https://archive.mozilla.org/pub/opus/opus-1.3.tar.gz
+        sha512: 438635667e72c326d6d06a30e0b3a5564ba6cf5d484066456e1118e1c2fe3a61b959d0afa77a1084b177b72f9230618b6b55c78bc4e24e9127a09fd345b42518
+
+  - name: opusfile
+    config-opts:
+      - --disable-doc
+      - --disable-examples
+    sources:
+      - type: archive
+        url: https://archive.mozilla.org/pub/opus/opusfile-0.11.tar.gz
+        sha512: ec3e282310cc4f25475c27b7bc8d1652dcb25d3ac6badf87bd1b4e5397fbe106a0dab81c60d88d198003a23d8a2c9bae8b661edc9b31433effeca438ce56a349
+
+  - name: taglib
+    buildsystem: cmake
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: http://taglib.org/releases/taglib-1.11.1.tar.gz
+        sha512: 7846775c4954ea948fe4383e514ba7c11f55d038ee06b6ea5a0a1c1069044b348026e76b27aa4ba1c71539aa8143e1401fab39184cc6e915ba0ae2c06133cb98
+
+  - name: wavpack
+    cleanup:
+      - /bin
+      - /share/man
+    sources:
+      - type: archive
+        url: http://www.wavpack.com/wavpack-5.3.0.tar.bz2
+        sha512: 2c5038abfbe7ab3b51fb30b3fc8b636117e9afe1821c40832fbdfb960d1153e0cb4a68dd84a89b205c3cdc10030d3aa7a7340b296d9c148e8847471c2e7c0cd1
+
+  - name: intltool
+    cleanup:
+      - "*"
+    sources:
+      - type: archive
+        url: https://launchpad.net/intltool/trunk/0.51.0/+download/intltool-0.51.0.tar.gz
+        sha256: 67c74d94196b153b774ab9f89b2fa6c6ba79352407037c8c14d5aeb334e959cd
+
+  - name: easytag
+    config-opts:
+      - --disable-Werror
+    post-install:
+      - mv org.gnome.EasyTAG.appdata.xml /app/share/appdata/$FLATPAK_ID.appdata.xml
+      - mv org.gnome.EasyTAG.desktop /app/share/applications/$FLATPAK_ID.desktop
+      - for i in $(find /app/share/icons/hicolor -name "easytag*"); do cp $i $(dirname $i)/$FLATPAK_ID.png; done;
+      - mv /app/share/icons/hicolor/symbolic/apps/$FLATPAK_ID.png /app/share/icons/hicolor/symbolic/apps/$FLATPAK_ID-symbolic.svg
+      - mv /app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.png /app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg
+      - rm /app/share/applications/easytag.desktop
+      - rm /app/share/appdata/easytag.appdata.xml
+      - rm /app/share/icons/hicolor/icon-theme.cache
```

Comment:
Why not use them?

Reply:
It should be pretty straightforward to fix them, the desktop file just needs the icon name change, while the appdata.xml is slightly more complicated in that it also needs the releases and content_rating sections in addition to the application name change. I'm just not familiar enough yet with `sed` and the like to just modify them, so I'll get to that in the next few days unless a patch file for both (like appdata.xml now I suppose) would be good enough.

Reply:
Also for the icon cache there's something about it not going to be exported in the output of the build so I thought it's not supposed to be there for Flatpak builds, I can take that out too if that's better.

Reply:
After commit cf062c5 the whole post-install section has been cleared out.